Add rename option to serializable_hash method (#48338)
rails/rails: activemodel/CHANGELOG.md

> *   `serializable_hash` has a new option called rename.

Reviewer: ```suggestion
*   `serializable_hash` and `as_json` have a new `:rename` option.
```

>     it can be used when you want to rename attributes or methods.

Reviewer: ```suggestion
    It can be used to rename generated hash keys for attributes or methods:
```

>     ```ruby
>     class Person
>       include ActiveModel::Serialization
> 
>       attr_accessor :name, :age
> 
>       def attributes
>         {'name' => nil, 'age' => nil}
>       end
>     end
> 
>     person.serializable_hash(rename: {name: :first_name}) # => {"first_name"=>"bob", "age"=>22}

Reviewer: As this is the changelog, I think we can be more terse and let the reader guess more context -- if it sounds interesting, they can check the documentation.

```suggestion
    ```ruby
    person.serializable_hash(only: %w(name age), rename: { name: :first_name })
      # => { "first_name" => "bob", "age" => 22 }
```

>     ```

Reviewer: ```suggestion
    ```

```
